Cubical/Foundations/Prelude.agda

```diff
@@ -201,6 +201,12 @@ substRefl px = transportRefl px
 funExt : {f g : (x : A) → B x} → ((x : A) → f x ≡ g x) → f ≡ g
 funExt p i x = p x i
 
+funExt2 : {A₀ : Type ℓ₀} {A₁ : Type ℓ₁} {B : A₀ → A₁ → Type ℓ₂}
```

Comment:
This is already defined here: https://github.com/agda/cubical/blob/master/Cubical/Functions/FunExtEquiv.agda#L38

Maybe you can add a comment pointing to this file here?